libs/api/domains/work-machines/src/lib/models/machineCategory.ts

```diff
@@ -0,0 +1,13 @@
+import { ObjectType, Field } from '@nestjs/graphql'
+
+@ObjectType()
```

Comment:
Need to prefix all object/input types with the api domain

`WorkMachinesMachineCategory` sounds silly so maybe `WorkMachineCategory`

See [docs](https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions)